languages/javascript/reference/keywords/var.md

```diff
@@ -1 +1,30 @@
 # `var`
+
+The `var` keyword declares a [variable](https://developer.mozilla.org/en-US/docs/Glossary/Variable), optionally initializing it to a value.
```

Comment:
This still doesn't link to either:

- Our JavaScript reference about variable (with that link somewhere in the doc)
- Global reference about variables.

Should use: `[title][reference]` link format.